images_config/ks-robot-master.cfg

```diff
@@ -211,6 +211,9 @@ echo "%users ALL=NOPASSWD:/usr/sbin/cob-shutdown" >> /etc/sudoers
 
 ################ NETWORK SETUP ################ 
 %post --interpreter=/bin/bash --logfile /root/ks-post.log
+
+apt-get purge modemmanager -y
```

Comment:
this line has nothing to do with the network setup

Reply:
previously it was in the apt-cacher section where it doesn't fit either. what does it do? where should we put it?

Reply:
I suggest before the apt-cacher configuration.

For some estrange reason the modemanager open automatically some devices and as the device is being used we can not launch the drivers, we had the problem with the bluetooth devices and the LEDs interface. You can prevent also the issue adding a udev rule with the attribute ENV{ID_MM_DEVICE_IGNORE}="1" , see https://github.com/ipa320/setup_cob4/blob/master/udev_rules/99-gripper.rules and https://github.com/ipa320/setup_cob4/blob/master/udev_rules/98-led.rules#L2, but I prefer the remove of the package. We don't need it and in case the udev rule changes or we have new devices we will not have the problem any more, the debug of the error was not trivial